### DISCUSSION TITLE
Can I change the "default_config_file_path"?

### DISCUSSION THREAD

**Discussion by zawasan23** [Snowflake-Labs/snowcli#957] [Q&A] [answered]
### Description

Change "default_config_file_path"

### Context

In "snow connection", you can specify a specific config.toml using --config-file,
I want to change the default path of "config.toml".

■ Default Path
```
Linux: ~/.config/snowflake/config.toml, but you can update it with XDG vars

Windows: %USERPROFILE%\AppData\Local\snowflake\config.toml

Mac: ~/Library/Application Support/snowflake/config.toml
```

It seems that I can change the default path of config.toml by changing the "default_config_file_path" displayed in "snow --info".
changing the "default_config_file_path" does not seem to be implemented yet, so I would appreciate it.
If anyone already knows how to do this, please let us know.

■ snow --info
```
{
    "key": "default_config_file_path",
    "value": <usr name>/AppData/Local/snowflake/config.toml"
},
```

**Comment by sfc-gh-turbaszek** [comment]
This is not possible at the moment. Using different config file requires you to specify `--config-file` every time. We do plan to address this issue.

**Comment by sfc-gh-pczajka** [comment, marked as answer]
Suggested workaround for now: set up `SNOWFLAKE_HOME` environment variable to a directory, which will contain `config.toml` file

**Comment by sfc-gh-turbaszek** [comment]
The `SNOWFLAKE_HOME` variable is the way to go. We cannot change config file using a config file entry. I'm going to close this issue and convert it to discussion for others reference
